### PR TITLE
feat: Add SchemaLookupCommand

### PR DESCRIPTION
**Pull request by hikchoi**
This PR:
- Adds `SchemaLookupCommand` with basic schema lookup functionality.
- Adds tests for basic schema lookup.

### CODE REVIEW

**Comment by hikchoi** [comment]
Checklist

- [x] Implement basic functionality
- [ ] Add tests for basic functionality

File: packages/plugin-core/package.json [hunk 1]
```diff
@@ -170,6 +170,12 @@
         "docLink": "dendron.topic.lookup.md",
         "docPreview": "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)"
       },
+      {
+        "command": "dendron.lookupSchemaV3",
```
Comment (kevinslin) [inline packages/plugin-core/package.json:174]: can we change the name to `Schema Lookup` so its consistent with the style of `Note Lookup`?

**Review by kevinslin — CHANGES_REQUESTED** [review]
iRight now when I select a schema, the picker seems to persist instead of hiding. Otherwise looks good. Is this meant to still be a draft?

File: packages/plugin-core/package.json (hunk 1 above)
Comment (hikchoi) [reply packages/plugin-core/package.json:174]: The title of `NoteLookupCommand` is `Lookup Note` so I was going for `Lookup Schema`, but that's already taken for lookupv2's schema lookup, went with the current one.

```
{
        "command": "dendron.lookup",
        "title": "Dendron: Lookup",
        ... // this is lookup v2
},
      {
        "command": "dendron.lookupNote",
        "title": "Dendron: Lookup Note",
        ... // this is NoteLookupCommand
      },
      {
        "command": "dendron.lookupSchema",
        "title": "Dendron: Lookup Schema",
        ... // this is lookup v2 with schema flavor
      },
      {
        "command": "dendron.lookupSchemaV3",
        "title": "Dendron: Lookup Schema V3",
        ... // this is SchemaLookupCommand
      },
```

Probably not a good idea to change v2 naming, so I would go for `Schema Lookup` and change `Lookup Note` to `Note Lookup`

**Comment by hikchoi** [comment]
> iRight now when I select a schema, the picker seems to persist instead of hiding. 
`noHidePickerOnAccept` was set to true for both `NoteLookup` and `SchemaLookup` for some reason. It's fixed now.

> Otherwise looks good. Is this meant to still be a draft?
Forgot to change it 😓

File: packages/plugin-core/package.json (hunk 1 above)
Comment (kevinslin) [reply packages/plugin-core/package.json:174]: ah, I see. 

in that case, feel free to delete
```
  LOOKUP_SCHEMA: {
    key: "dendron.lookupSchema",
    title: `${CMD_PREFIX} Lookup Schema`,
    group: "navigation",
    keybindings: {
      mac: "cmd+shift+L",
      key: "ctrl+shift+l",
      when: DendronContext.PLUGIN_ACTIVE,
    },
    desc: "Initiate schema lookup",
    docLink: "dendron.topic.lookup.md",
    docPreview:
      "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)",
  },
```

and update lookup schema to have the new keyboard shortcut. the plan is to replace it and schema lookup should be at full functionality now.

File: packages/plugin-core/src/constants.ts [hunk 2]
```diff
@@ -281,14 +281,14 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
   },
   LOOKUP_NOTE: {
     key: "dendron.lookupNote",
-    title: `${CMD_PREFIX} Lookup Note`,
+    title: `${CMD_PREFIX} Note Lookup`,
     group: "navigation",
     desc: "Initiate note lookup",
     when: DendronContext.PLUGIN_ACTIVE,
   },
   LOOKUP_SCHEMA_V3: {
```
Comment (kevinslin) [inline packages/plugin-core/src/constants.ts:289]: don't forget to delete and to add the keyboard shortcuts back to the new lookup schema

```
  LOOKUP_SCHEMA: {
    key: "dendron.lookupSchema",
    title: `${CMD_PREFIX} Lookup Schema`,
    group: "navigation",
    keybindings: {
      mac: "cmd+shift+L",
      key: "ctrl+shift+l",
      when: DendronContext.PLUGIN_ACTIVE,
    },
    desc: "Initiate schema lookup",
    docLink: "dendron.topic.lookup.md",
    docPreview:
      "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)",
  },
```

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: packages/plugin-core/src/constants.ts (hunk 2 above)
Comment (hikchoi) [reply packages/plugin-core/src/constants.ts:289]: are we already deprecating v2?

Comment (hikchoi) [reply packages/plugin-core/src/constants.ts:289]: reading the comments backwards 😅  just saw the other comment.

**Comment by hikchoi** [comment]
@kevinslin removed the command key `LOOKUP_SCHEMA_V3` and used `LOOKUP_SCHEMA` for v3, un-subbed v2 schema lookup command, and reverted changes to lookup v3 command titles back (`NoteLookupCommand` is back to `Lookup Note` and `SchemaLookupCommand` is back to `Lookup Schema`).

> and update lookup schema to have the new keyboard shortcut. the plan is to replace it and schema lookup should be at full functionality now.

That's what you meant here right? 😅

**Comment by kevinslin** [comment]
> @kevinslin removed the command key `LOOKUP_SCHEMA_V3` and used `LOOKUP_SCHEMA` for v3, un-subbed v2 schema lookup command, and reverted changes to lookup v3 command titles back (`NoteLookupCommand` is back to `Lookup Note` and `SchemaLookupCommand` is back to `Lookup Schema`).
> 
> > and update lookup schema to have the new keyboard shortcut. the plan is to replace it and schema lookup should be at full functionality now.
> 
> That's what you meant here right? 😅

yep. One last thing -> can you add the keyboard shortcut for schema to the new schema lookup command?

**Comment by hikchoi** [comment]
@kevinslin Since I'm reusing the `LOOKUP_SCHEMA` constant, it will use the same keyboard shortcut cmd/ctrl + shift + l
see [here](https://github.com/dendronhq/dendron/blob/783ba252540426d9fdd2f6f1469c758dfe41026c/packages/plugin-core/src/constants.ts#L324)

I did find an inconsistent quick pick title just now and pushed 😓

**Review by kevinslin — APPROVED** [review]

**Pull request merged**